### PR TITLE
Polish translation improvement

### PR DESCRIPTION
**Pull request by w1stler**
Minor improvement in wording.

### CODE REVIEW

**Comment by outsideris** [comment]
@ImJustToNy Could you confirm this?

File: locales/pl.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "HEADER": "Podaj swój email pod spodem aby dołączyć do %s na Slack!",
+  "HEADER": "Podaj swój adres email, aby dołączyć do %s na Slacku!",
```
Comment (ImJustToNy) [inline locales/pl.json:2]:
```suggestion
  "HEADER": "Podaj swój adres email, aby dołączyć do %s na Slack'u!",
```

File: locales/pl.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   "ENTER_EMAIL": "Podaj swój adres email",
   "ENTER_TOKEN": "Podaj swój kod zaproszenia",
-  "TITLE": "Dołącz do społeczności %s na Slack!"
+  "TITLE": "Dołącz do społeczności %s na Slacku!"
```
Comment (ImJustToNy) [inline locales/pl.json:5]:
```suggestion
  "TITLE": "Dołącz do społeczności %s na Slack'u!"
```

**Review by ImJustToNy — COMMENTED** [review]
I believe it looks better as "Slack'u" not "Slacku", but it is discussable.

Ref: https://sjp.pwn.pl/poradnia/haslo/apostrof-w-wyrazach-obcych;11890.html

File: locales/pl.json (hunk 2 above)
Comment (w1stler) [reply locales/pl.json:5]: Disagree.

You add apostrophe, when the word is ending with vowel, eg. iPhone'a, Google'a (and this kind of examples are present in the link you've provided). In other situations you skip it, eg. Facebooka, Slacka, Androida. Writing everyting with the apostrophe would look... weird. Have a look:

"Włączyłem telefon z Android'em i uruchomiłem Slack'a. Na MacBook'u wyłączyłem Twitter'a."

Comment (ImJustToNy) [reply locales/pl.json:5]: Yeah, you're right. Leave it as it is.

**Comment by ImJustToNy** [comment]
Safe to merge I believe.

@outsideris

**Comment by outsideris** [comment]
@ImJustToNy Thank you.

**Pull request merged**